### ISSUE TITLE
Browser Chromium based and MathJax library

### ISSUE THREAD

**Issue by sharkboyto** [nvaccess/nvda#13475]
### Steps to reproduce:
- with Google Chrome or Edge open: [](https://mathjax.github.io/MathJax-demos-web/equation-numbers.html)
### Actual behavior:
- the mathematical formula is correctly rendered, but before it, there is an unrendered formula (plain text), a duplicate, which shouldn't be there.
### Expected behavior:
- the formula must be written only once, rendered in mathml;
- in Mozilla Firefox everything works fine.
#### NVDA installed/portable/running from source:
installed and portable
#### NVDA version:
2021.3.1
#### Windows version:
Versione	10.0.19044 Build 19044

#### Name and version of other software in use when reproducing the issue:
Math Player 4.0 and the latest Chrome and Edge browser versions
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes: 2019.3 and 2020.1
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
Thanks @sharkboyto - would you be able to provide a sample [codepen](https://codepen.io/)?

**Comment by Steve-Noble** [comment]
This is the identical issue I raised on January 6 #13218
For samples and additional details please see that issue.
I do hope there will be some resources devoted to fixing this issue soon. Mathematics on the web is basically broken for anyone trying to use NVDA with either Chrome or Edge.

**Comment by Adriani90** [comment]
Closing as duplicate of #13218.
